src/main/webapp/index.html

```diff
@@ -22,71 +22,33 @@
       <button onclick="getClassID()">Get class ID</button>
       <br /><br />
 
-      <textarea name="class-input">Write opinion on class here.</textarea>
+      <textarea name="class-input" maxlength="1500">Write opinion on class here.</textarea>
       <br /><br />
 
       <p>How would you rate this class?</p>
-      <input
-        type="radio"
-        id="excellent"
-        name="rating-class"
-        value="excellent"
-      />
-      <label for="excellent">Excellent.</label>
-      <input
-        type="radio"
-        id="acceptable"
-        name="rating-class"
-        value="acceptable"
-      />
-      <label for="acceptable">Acceptable.</label>
-      <input type="radio" id="neutral" name="rating-class" value="neutral" />
-      <label for="neutral">Neutral.</label>
-      <input type="radio" id="not-good" name="rating-class" value="not-good" />
-      <label for="not-good">Not good.</label>
-      <input type="radio" id="terrible" name="rating-class" value="terrible" />
-      <label for="terrible">Terrible.</label>
+      <input type="rating-class" id="rating-class" name="rating-class" min="0" max="5" />
+      <label for="rating-class">Class Rating (From 0 = Terrible to 5 = Excellent)</label>
+      <br><br>
```

Comment:
Now that you're out of the prototype phase I'd recommend switching from using multiple br elements to adding padding/margin via css.

(Also your br tags are inconsistently styled.)